commons/src/main/java/org/infinispan/commons/marshall/AdminFlagExternalizer.java

```diff
@@ -0,0 +1,31 @@
+package org.infinispan.commons.marshall;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.Set;
+
+import org.infinispan.commons.api.CacheContainerAdmin;
+
+public class AdminFlagExternalizer extends AbstractExternalizer<CacheContainerAdmin.AdminFlag> {
+   @Override
+   public void writeObject(ObjectOutput output, CacheContainerAdmin.AdminFlag flag) throws IOException {
+      output.writeUTF(flag.name());
```

Comment:
Maybe we should use `ordinal` and `values()[offset]` instead of writing the entire name? And actually do you find the Externalizer for enums, are they worth it still?

Reply:
ok